h2o-core/src/main/java/water/jdbc/SQLManager.java

```diff
@@ -43,11 +48,12 @@
      SELECT * FROM (
         SELECT ROW_NUMBER() OVER () AS RowNum_, <table>.* FROM <table>
      ) QUALIFY RowNum_ BETWEEN x and x+y;
-    */
-    String db_sys = connection_url.split(":",3)[1];
-    final boolean needFetchClause = db_sys.equals("oracle") || db_sys.equals("sqlserver");
-    int catcols = 0, intcols = 0, bincols = 0, realcols = 0, timecols = 0, stringcols = 0; 
-    final int numCol; 
+     */
+    String databaseType = connection_url.split(":", 3)[1];
+    initializeDatabaseDriver(databaseType);
```

Comment:
These checks are originally case-sensitive. Maybe db_sys should be converted to lower case to make any ongoing matching case-insensitive.